client/src/pages/Start.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import StartAnimation from '../components/StartAnimation';
+
+let timeout;
+
+function Start() {
+  clearTimeout(timeout);
+  timeout = setTimeout(function () {
+    window.location = '/Welcome';
+  }, 4000);
+
+  return <StartAnimation />;
+}
```

Comment:
What is the reason why you defined `timeout` outside of this component?
This would be a good case for a `useEffect`.

```suggestion

function Start() {
  React.useEffect(() => {
    const timeoutId = setTimeout(function () {
      window.location = '/Welcome';
    }, 4000);
    
    return () => {
      clearTimeout(timeoutId);
    };
  }, []);
  
  return <StartAnimation />;
}
```

Reply:
It actually didn't work inside the function, hence I had to define it outside.